### PR TITLE
key-value Makefiles

### PR DESCRIPTION
**Pull request by dungwinux**

### CODE REVIEW

File: key-value/cpp/Makefile [hunk 1]
```diff
@@ -1,28 +1,9 @@
...
-
-clean:
-	rm key_value-o2.o
-	rm key_value-o3.o
-	rm key_value-debug.o
-	rm key_value-o0.o
-	rm micro_hashmap.o
+run:
+	screen -dmS key_value ./key_value.o 127.0.0.1 51234 24576 1024
+	$(REDIS_BENCHMARK) -c $(JOBS) -n 2000000 -d 64 -r 500000 -t set,get -h 127.0.0.1 -p 51234
```
Comment (nicovank) [inline key-value/cpp/Makefile]: Definitely set `JOBS` to some default or throw if not set.

IMO just make this `redis-benchmark`.

File: key-value/go/Makefile [hunk 2]
```diff
@@ -1,11 +1,11 @@
-.PHONY: all clean
+ifndef GO
+GO := go
+endif
 
-GO = $(LANGBENCH)/runtimes/go/go-1.14.1-vanilla/go
+compile:
+	$(GO) build -o key_value.o
```
Comment (nicovank) [inline key-value/go/Makefile]:
```suggestion
	$(GO) build -o key_value.o key_value.go hashmap.go
```

I think you need the filenames to be able to compile without creating a module etc.

**Review by nicovank — APPROVED** [review]
I also only have C++ and Go working at the moment.

File: key-value/js/Makefile [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+ifndef JOBS
+JOBS := 8
+endif
+
+compile:
+
+run:
+	screen -dmS key_value $(NODE) key_value.js 127.0.0.1 51234 24576 1024
+	redis-benchmar -c $(JOBS) -n 2000000 -d 64 -r 500000 -t set,get -h 127.0.0.1 -p 51234
```
Comment (nicovank) [inline key-value/js/Makefile]:
```suggestion
	redis-benchmark -c $(JOBS) -n 2000000 -d 64 -r 500000 -t set,get -h 127.0.0.1 -p 51234
```

**Review by nicovank — APPROVED** [review]
Java still fails on my machine, tried to debug but not sure what is going on. This is also hard because we don't work with redis-benchmark from source.

I am not 100% sure yet how we will measure this either since it's two separate things really. I think we should just merge this for now, and possibly fix the rest later.

**Comment by dungwinux** [comment]
> Java still fails on my machine, tried to debug but not sure what is going on. This is also hard because we don't work with redis-benchmark from source.
> 
> I am not 100% sure yet how we will measure this either since it's two separate things really. I think we should just merge this for now, and possibly fix the rest later.

Okay, I will merge it now

**Pull request merged**